utils/network.py

```diff
@@ -214,6 +290,8 @@ def construct_network_graph(
         dfs: dataframes in the order: inds_df, orgs_df, transactions_df
 
     Returns:
+        creates a network metrics file, plots a network graph,
```

Comment:
this is misleading -- the function doesn't return anything. Use the body of the docstring to describe what the function does.